airtime_mvc/application/common/ProvisioningHelper.php

```diff
@@ -182,5 +215,16 @@ private function initializeMusicDirsTable($instanceId)
         $musicDir->save();
     }
 
+    /**
+     * Initialize preference values passed from the dashboard (if any exist)
+     */
+    private function initializePrefs() {
+        if ($this->statio_name) {
+            Application_Model_Preference::SetStationName($this->station_name);
+        }
+        if ($this->descption) {
+            Application_Model_Preference::SetStationDescription($this->description);
+        }
+    }
```

Comment:
This code should be in ProvisioningHelper, not in the controller class. (it's "business logic")